bundles/azure-service-broker-0.0.1/chart/azure-service-broker/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+appVersion: 1.3.1
+description: A Helm chart for Open Service Broker For Azure
+home: https://github.com/azure/open-service-broker-azure
+keywords:
+- azure
+- services
+- service broker
+name: azure-service-broker
+sources:
+- https://github.com/azure/open-service-broker-azure
+- https://hub.docker.com/r/microsoft/azure-service-broker/
+tillerVersion: '>=2.7.0-rc1'
+version: 1.3.1
```

Comment:
1.0.0 if we treat that as our chart, or 1.4.0

Reply:
Do you mean `0.0.1` version? (GCP has version 0.0.1 so I maintain consistency)

Reply:
ok, lets set 0.0.1

Reply:
👍